### PR TITLE
fix(eventual-send): Remove ?. and ?? operators for RESM

### PR DESCRIPTION
**Pull request by kriskowal**
This allows eventual-send to be used in -r esm style Agoric deploy scripts.

### CODE REVIEW

File: packages/eventual-send/src/track-turns.js [hunk 1]
```diff
@@ -17,10 +17,10 @@ let hiddenCurrentEvent = 0;
 
 // TODO Use environment-options.js currently in ses/src after factoring it out
 // to a new package.
-const env = globalThis?.process?.env;
+const env = ((globalThis.process || {}).env ||{});
```
Comment (mhofman) [inline packages/eventual-send/src/track-turns.js:20]: Is this linting?

Comment (kriskowal) [reply packages/eventual-send/src/track-turns.js:20]: No, pushed update.

Comment (mhofman) [reply packages/eventual-send/src/track-turns.js:20]: Still seeing the same, bad push?

Comment (kriskowal) [reply packages/eventual-send/src/track-turns.js]: Yes, addressed.

**Review by mhofman — APPROVED** [review]
Simple enough

**Pull request merged**